modules/proc_deploy-quay-openshift-operator.adoc

```diff
@@ -1081,6 +1204,63 @@ Environment variables for the Quay configuration pod can be managed by specifyin
 User defined environment variables are given precedence over those managed by the operator. Undesirable results may occur if conflicting keys are used.
 ====
 
+= Configuring {productname} (post-deployment)
+
+After the Quay Operator deploys {productname}, by default the Config Tool
+continues to run. Going forward, you can use the Config Tool or the
+Quay Operator itself to update and maintain your {productname} deployment.
+
+== Using the Config Tool
+The {productname} Config Tool provides a web UI for enabling or
+modifying many of the settings in your {productname} cluster. 
+To use the Config Tool:
+
+. Get the route to the Config Tool by typing:
++
+```
+$ oc get route 
+NAME  HOST/PORT                                      PATH SERVICES PORT TERMINATION WILDCARD
+...   example-quayecosystem-quay-config.example.com  ...
+```
+. Add `https://` to the HOST/PORT entry for the Config Tool and enter it into your
+web browser.
+
+. When prompted, log in using the Config Tool user name and password
+(`quayconfig` and `quay`, by default).
+
+. Select `Modify configuration for this cluster`.
+
+At this point you can change the configuration as you choose.
+When you are done, select Save Configuration Changes. Here are a few
+things you should know about using the Config Tool:
```

Comment:
There is a new feature about continue editing after validation: [PROJQUAY-560](https://issues.redhat.com/browse/PROJQUAY-560). Maybe we can update some items for it.

Reply:
Good idea. Done.